pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -371,12 +371,9 @@ def get_interface_nodes(self, time_slice=0):
         >>> dbn.get_interface_nodes()
         [('D', 0)]
         """
-        if not isinstance(time_slice, int) or time_slice < 0:
```

Comment:
@FelipeGiro I am not sure if this check for the input argument needs to be removed. Am I missing something?